todo.txt

```diff
@@ -0,0 +1,5 @@
++ support different types of spans: client span (new span ID), rpc server span (same span ID) , (component, and lc) local span (new span ID, different zipkin annotations)
```

Comment:
profile performance of Noop tracer vs. Jaeger tracer with const sampling off and on